webapp/app/js/filters.js

```diff
@@ -39,11 +39,21 @@ treeherder.filter('stripHtml', function() {
 
 treeherder.filter('linkifyBugs', function() {
     return function(input) {
-        var re = new RegExp('(?:Bug (\\d+))', 'ig');
-        var str = input || '';
-        return str.replace(re,
-            '<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=$1" target="_blank" title="Bug $1">$1</a>'
-        );
+        if (input) {
+            var re = /.*-- (bug-)?([0-9]+)/i; //regex for matching bug number
+            var bug_matches = re.exec(input);
```

Comment:
Let's just do something like var bug_matches = /PR#([0-9]+)/i.exec(input) instead of defining (and then redefining) a temporary re variable.